courses/advanced-foundry/6-account-abstraction/2-code-overview/+page.md

```diff
@@ -0,0 +1,25 @@
+# Account Abstraction Lesson 2: Code Overview
+
+Welcome to lesson 2 of the Account Abstraction course! In this lesson, we will take a look at the codebase for Ethereum and zkSync. They are minimally structured and serve as a great starting point for learning about account abstraction.
```

Comment:
Can we modify it to "These are minimal accounts" rather than "They are minimally structured"